tools/bazar/presentation/javascripts/form-edit-template.js

```diff
@@ -563,6 +563,7 @@ var yesWikiTypes = {
   listedatedeb: { type: "date" },
   listedatefin: { type: "date" },
   jour: { type: "date" },
+  carte_google: { type: "map" },
```

Comment:
```suggestion
  map: { type: "carte_google" },
```

et normalement tout fonctionne.

Reply:
Dès que cette modification est faite, OK pour moi (les autres modifs c'est comme tu le sens :-))

Reply:
c'est pas map: { type: "map" }, plutot?